pom.xml

```diff
@@ -95,6 +95,18 @@
       <artifactId>gson</artifactId>
       <version>2.8.2</version>
     </dependency>
+    <dependency>
+      <!-- bundle: com.google.cloud.tools.login.plugins-login-common -->
+      <groupId>com.google.cloud.tools.login</groupId>
+      <artifactId>plugins-login-common</artifactId>
+      <version>0.2.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-oauth2</artifactId>
```

Comment:
I see this one is not OSGified (no OSGi info in the JAR metadata), so yeah, this won't work, and it explains the error, right?

Reply:
Makes sense. Let me get on that. We may need to do this in a different order of fixes.

Reply:
b/116784645